mod/ping.php

```diff
@@ -123,11 +123,10 @@ function ping_init(&$a) {
 		$mail = count($mails);
 
 		if ($a->config['register_policy'] == REGISTER_APPROVE && is_site_admin()){
-			$regs = q("SELECT `contact`.`name`, `contact`.`url`, `contact`.`micro`, `register`.`created`, COUNT(*) as `total` FROM `contact` RIGHT JOIN `register` ON `register`.`uid`=`contact`.`uid` WHERE `contact`.`self`=1");
-			if ($regs)
-				$register = $regs[0]['total'];
+			$regs = q("SELECT `contact`.`name`, `contact`.`url`, `contact`.`micro`, `register`.`created` FROM `contact` RIGHT JOIN `register` ON `register`.`uid`=`contact`.`uid` WHERE `contact`.`self`=1");
+			$register = count($regs);
```

Comment:
It's already used in line 254, not sure where or how you want to use it. The result count directly after query is already used in lines 114 and 123, should they be protected by is_result too?